source/_patterns/04-components/tag/_tag.scss

```diff
@@ -1,10 +1,10 @@
 .tag {
   @include link(gesso-color(ui, generic, accent-dark));
-  @include vertical-spacing(rem(gesso-spacing(1)));
   border: 2px solid gesso-color(ui, generic, accent-light);
   border-radius: 70px;
   display: inline-block;
   font-size: rem(gesso-font-size(1));
+  margin: 0 0 rem(gesso-spacing(1)) 0;
```

Comment:
[minor nitpick]

Another prettier miss? This should be `margin: 0 0 rem(gesso-spacing(1));`